web/src/App.js

```diff
@@ -326,23 +326,29 @@ class FiltersFormBox extends React.Component {
         <Route exact path="/" />
         <Route path="/:index/change/" />
         <Route path="/:index/repos/">
-          <CFiltersForm />
+          <CFiltersForm index={this.props.index} />
```

Comment:
When the Route capture an index, we can use the `component` property to `CFiltersForm`, similarly to what is done in the App's Route.

Reply:
yes probably, the underlying query to the project endpoint was performed with undefined value.